spec/models/creator_spec.rb

```diff
@@ -50,6 +50,25 @@
       expect(response['response']['docs'].first['creator_tesim']).not_to include "McGrattan, Ellen R."
     end
   end
+  context 'with an accidentally-deleted creator' do
+    let(:work) {
+      FactoryBot.build(:populated_publication,
+      creator_id: [creator_one.authority_rdf, creator_two.authority_rdf, creator_three.authority_rdf])
+    }
+    let(:solr) { Blacklight.default_index.connection }
+    let(:creator_one) { FactoryBot.create(:creator, display_name: 'Kehoe, Patrick J.') }
+    let(:creator_two) { FactoryBot.create(:creator, display_name: 'Backus, David', alternate_names: ['Backus, Davey', 'Backus-Up, David']) }
+    let(:creator_three) { FactoryBot.create(:creator, display_name: 'Kehoe, Timothy J.') }
+
+    it "fails gracefully" do
+      creator_one
+      creator_two
```

Comment:
It seems surprising to have to call `creator_one` explicitly - does the test fail if you comment out line #64-65 and just rely on the invocation in line #56?